README.md

```diff
@@ -47,15 +100,14 @@ make_idl_files.py MTM1M3 MTM1M3TS MTMount MTVMS
 
 ## SALComm
 
-Heart of the application is SALComm. The module links ts_salobj callbacks with
-Qt Signals. Names of signals matches SAL events and telemetry topics. This
-allows for simple integration of DDS/SAL and GUI widgets. Widgets in need to
-receive SAL data accept SALComm as constructor parameter, and after setting up
-the widget SALComm provided Qt Signals are connected to slots in the widget.
+Heart of the application is SALComm.
+The module links ts_salobj callbacks with Qt Signals.
+Names of signals matches SAL events and telemetry topics.
+This allows for simple integration of DDS/SAL and GUI widgets.
+Widgets in need to receive SAL data accept SALComm as constructor parameter, and after setting up the widget SALComm provided Qt Signals are connected to slots in the widget.
 
 ## Custom widgets
 
 Qt Slots are decorated with @Slot and usually not documented, as the only
-functions is to update widgets with data received from SAL/DDS. Please see
-[PySide2 documentation](https://wiki.qt.io/Qt_for_Python_Signals_and_Slots) and
-[SALComm](SALComm.py) for details how this works.
+functions is to update widgets with data received from SAL/DDS.
+Please see [PySide2 documentation](https://wiki.qt.io/Qt_for_Python_Signals_and_Slots) and [SALComm](SALComm.py) for details how this works.
```

Comment:
This is the problem wrapping removes - too wide chars. Please keep the original text, so we can see diffs.

Reply:
Updated in the commit: 535c3c1. Thanks!